pyomo/solvers/plugins/solvers/CPLEX.py

```diff
@@ -159,7 +159,7 @@ def _warm_start(self, instance):
             mst_file.write("<header/>\n")
             mst_file.write("<quality/>\n")
             mst_file.write("<variables>\n")
-            for var in instance.component_data_objects(Var, active=True):
+            for var in instance.component_data_objects(Var):
```

Comment:
Why do we need to loop through variables that are not active?

Reply:
It turns out that Vars aren't active (they are not ActiveComponents and cannot be deactivated).  The active flag only controlled if the generator descended into all blocks or only active blocks.  This change allows the generator to descend into all blocks - not just the active ones.

Reply:
Right. (We just had this conversation :)